ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -118,13 +118,16 @@ type wireframeBase = {
     /**
      * @description x and y are the top left corner of the element, if they are present then the element is absolutely positioned, if they are not present this is equivalent to setting them to 0
      */
-    x: number
-    y: number
+    x?: number
+    y?: number
     /*
-     * @description width and height are the dimensions of the element, the only accepted units is pixels. You can omit the unit.
+     * @description the width dimension of the element, the only accepted units is pixels. You can omit the unit. If not received it is not set
      */
-    width: number
-    height: number
+    width?: number
+    /*
+     * @description the height dimension of the element, the only accepted units is pixels. You can omit the unit. If not received it is not set
+     */
+    height?: number
```

Comment:
this feels the nicest solution here...

don't want to set a width or height, that's fine... if it's not a component where we're expecting that then the behaviour is unspecified

CSS will do its thing and stuff might work.

Alternatively, these remain required but '100vw' becomes a valid value for width????

Reply:
no, even typing that out makes me realise it's silly